hazelcast-sql/src/test/java/com/hazelcast/jet/sql_slow/JetMapScanMigrationStressTest.java

```diff
@@ -90,6 +90,23 @@ public void stressTest_hash() throws InterruptedException {
         assertThat(mutatorException.get()).isNull();
     }
 
+    @Test(timeout = 600_000)
+    public void stressTest_fullFetch() throws InterruptedException {
```

Comment:
I'd name the three tests in this class this way and would add them in this order:
- `stressTest_noIndex`
- `stressTest_hashIndex`
- `stressTest_sortedIndex`